packages/core/test/testers.test.ts

```diff
@@ -1,213 +1,233 @@
-            foo: { type: 'string' }
-        }
-    };
-    const uischema: ControlElement = {
-        type: 'Control',
-        scope: {
-            $ref: '#/properties/bar'
-        }
-    };
-    t.false(schemaMatches(subSchema => false)(uischema, schema));
+  const schema: JsonSchema = {
+    type: 'object',
+    properties: {
+      foo: { type: 'string' }
+    }
+  };
+  const uischema: ControlElement = {
+    type: 'Control',
+    scope: {
+      $ref: '#/properties/bar'
+    }
+  };
+  t.false(schemaMatches(subSchema => false)(uischema, schema));
 });
 
 test('refEndsWith checks whether the ref of a control ends with a certain string', t => {
-    const uischema: ControlElement = {
-        type: 'Control',
-        scope: {
-            $ref: '#/properties/bar'
-        }
-    };
-    t.true(refEndsWith('properties/bar')(uischema, undefined));
+  const uischema: ControlElement = {
+    type: 'Control',
+    scope: {
+      $ref: '#/properties/bar'
+    }
+  };
+  t.true(refEndsWith('properties/bar')(uischema, undefined));
 });
 
 test('refEndsWith should return false for non-control UI schema elements', t => {
-    const label: LabelElement = {
-        type: 'Label',
-        text: 'some text'
-    };
-    t.false(refEndsWith('properties/bar')(label, undefined));
+  const label: LabelElement = {
+    type: 'Label',
+    text: 'some text'
+  };
+  t.false(refEndsWith('properties/bar')(label, undefined));
 });
 
 test('refEndIs checks whether the last segment a control ref equals a certain string', t => {
-    const uischema: ControlElement = {
-        type: 'Control',
-        scope: {
-            $ref: '#/properties/bar'
-        }
-    };
-    t.true(refEndIs('bar')(uischema, undefined));
+  const uischema: ControlElement = {
+    type: 'Control',
+    scope: {
+      $ref: '#/properties/bar'
+    }
+  };
+  t.true(refEndIs('bar')(uischema, undefined));
 });
 
 test('refEndIs should return false for non-control UI schema elements', t => {
-    const label: LabelElement = {
-        type: 'Label',
-        text: 'some text'
-    };
-    t.false(refEndIs('bar')(label, undefined));
+  const label: LabelElement = {
+    type: 'Label',
+    text: 'some text'
+  };
+  t.false(refEndIs('bar')(label, undefined));
 });
 
 test('and should allow to compose multiple testers', t => {
-    const schema: JsonSchema = {
-        type: 'object',
-        properties: {
-            foo: { type: 'string' }
-        }
-    };
-    const uischema: ControlElement = {
-        type: 'Control',
-        scope: {
-            $ref: '#/properties/foo'
-        }
-    };
-    t.true(and(
-        schemaTypeIs('string'),
-        refEndIs('foo')
-    )(uischema, schema));
+  const schema: JsonSchema = {
+    type: 'object',
+    properties: {
+      foo: { type: 'string' }
+    }
+  };
+  const uischema: ControlElement = {
+    type: 'Control',
+    scope: {
+      $ref: '#/properties/foo'
+    }
+  };
+  t.true(and(
+    schemaTypeIs('string'),
```

Comment:
Offtopic, but I think we should rename `schemaTypeIs` and `schemaMatches` as we actually mean the schema referenced by the control. Do you agree? If so, we should rename these in a follow-up PR.

Reply:
Yes I agree, will you open an Issue for this?

Reply:
Yes, done, see #734